src/SFA.DAS.Charities.Api/SFA.DAS.Charities.Api.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning" Version="3.1.6" />
```

Comment:
Missing Application Insights. Make sure this is there for Dev-Test environments

Reply:
Added